Source/SteveSloth/Public/MyEnemyAIIdleState.h

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************************
+* Copyright: SteveSloth
+* Name: Aidan MacLeod
+* Script: MyEnemyAIIdleState.h
+* Date: May 6. 2024
+* Description: Idle State Class for all enemies to inherit from
+* TODO:
+* Known Bugs:
+****************************************************************************************/
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Character.h"
+#include "Kismet/GameplayStatics.h"
+#include "MyEnemyBaseState.h"
+#include "MyEnemyBaseClass.h"
+#include "MyLevelManager.h"
+#include "MyEnemyAIIdleState.generated.h"
```

Comment:
get rid of all the includes you're not using. the way you're using this causes circular dependencies